opensubtitles/discover_test.go

```diff
@@ -0,0 +1,13 @@
+package opensubtitles
+
+import (
+	"testing"
+)
+
+func TestDiscoverService_MostDownloaded(t *testing.T) {
```

Comment:
Codacy found an issue: [parameter 't' seems to be unused, consider removing or renaming it as _](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)